loopdb/migration_02_swap_publication_deadline.go

```diff
@@ -134,33 +33,26 @@ func migrateSwapPublicationDeadline(tx *bbolt.Tx, chainParams *chaincfg.Params)
 		}
 
 		// With the main swap bucket obtained, we'll grab the
-		// raw swap contract bytes and decode it.
+		// raw swap contract bytes.
 		contractBytes := swapBucket.Get(contractKey)
 		if contractBytes == nil {
 			return errors.New("contract not found")
 		}
 
-		// Read the old format....
-		contract, err := deserializeLoopOutContractV01(
-			contractBytes, chainParams,
-		)
-		if err != nil {
+		// Write the current contract serialization into a buffer.
+		b := &bytes.Buffer{}
```

Comment:
Can't this just be `b := bytes.NewBuffer(contractBytes)`? Should be ok since we only read from `contractBytes` later. Or is this considered bad practice?

Reply:
I'm not sure tbh... The docs say
```
NewBuffer is intended to prepare a Buffer to read existing data. It can also be used to set the initial size of the internal buffer for writing. To do that, buf should have the desired capacity but a length of zero.
```
maybe safest to avoid it?